packages/lesswrong/components/users/DialogueSuggestionsPage.tsx

```diff
@@ -0,0 +1,782 @@
+      {userData?.biography?.plaintextDescription }
+    </div>
+  )
+};
+
+const UserPostsYouveRead = ({ classes, targetUserId }: { classes: ClassesType, targetUserId: string, components:ComponentTypes }) => {
+  const currentUser = useCurrentUser();
+
+  const { loading, error, data } = useQuery(gql`
+    query UsersReadPostsOfTargetUser($userId: String!, $targetUserId: String!) {
+      UsersReadPostsOfTargetUser(userId: $userId, targetUserId: $targetUserId) {
+        _id
+        title
+      }
+    }
+  `, {
+    variables: { userId: currentUser?._id, targetUserId: targetUserId },
+  });
+
+  const readPosts:[DbPost] = data?.UsersReadPostsOfTargetUser
+
+  const readPostsContainerRef = useRef<HTMLDivElement | null>(null);
+  const { isScrolledToTop, isScrolledToBottom } = useScrollGradient(readPostsContainerRef);
+
+  if (loading) return <p>Loading...</p>;
+  if (error) return <p>Error: {error.message}</p>;
+
+  return (
+    <div className={`${classes.gradientBigTextContainer} ${isScrolledToTop ? 'scrolled-to-top' : ''} ${isScrolledToBottom ? 'scrolled-to-bottom' : ''}`} ref={readPostsContainerRef}>
+      {readPosts.length > 0 ? (
+        readPosts.slice(0, 8).map((post, index) => {
+            return (
+              <PostsTooltip key={index} postId={post._id}>
+                <a key={index} href={"https://www.lesswrong.com/posts/gDijQHHaZzeGrv2Jc/"+post._id}>• {post.title}</a>
+                <br/>
+              </PostsTooltip>
+            )
+          })
+      ) : (
+        <p>(no posts read...)</p>
+      )}
+    </div>
+  );
+};
+
+const Headers = ({ titles, className }: { titles: string[], className: string }) => {
+  return (
+    <>
+      {titles.map((title, index) => (
+        <h5 key={index} className={className}>{title}</h5>
+      ))}
+    </>
+  );
+};
+
+const DialogueCheckBox: React.FC<{
+  targetUserId : string;
+  targetUserDisplayName : string;
+  checkId?: string;
+  isChecked: boolean, 
+  isMatched: boolean;
+  classes: ClassesType;
+}> = ({ targetUserId, targetUserDisplayName, checkId, isChecked, isMatched, classes}) => {
+  const currentUser = useCurrentUser();
+  const { captureEvent } = useTracking(); //it is virtuous to add analytics tracking to new components
+
+  const [localChecked, setLocalChecked] = useState(false);
+  const [upsertDialogueCheck] = useMutation(gql`
+    mutation upsertUserDialogueCheck($targetUserId: String!, $checked: Boolean!) {
+      upsertUserDialogueCheck(targetUserId: $targetUserId, checked: $checked) {
+          _id
+          __typename
+          userId
+          targetUserId
+          checked
+          match
+        }
+      }
+    `)
+
+  async function handleNewMatchAnonymisedAnalytics() {
+    captureEvent("newDialogueReciprocityMatch", {}) // we only capture match metadata and don't pass anything else
```

Comment:
What is the match metadata that's captured (which presumably excludes any sensitive information apart from the identity of the user who just had a match)?

Reply:
No data. Just the fact that there was a match and the associated context.